src/response/v2.rs

```diff
@@ -0,0 +1,381 @@
+        let progress = tokens[3].clone().to_uint().unwrap();
+        let is_finalized = tokens[4].clone().to_bool().unwrap();
+        let proposed_value = tokens[5].clone().to_address().unwrap();
+        let contract_type = tokens[6].clone().to_uint().unwrap();
+        let creator = tokens[7].clone().to_address().unwrap();
+        let memo = tokens[8].clone().to_string().unwrap();
+        let can_be_finalized_now = tokens[9].clone().to_bool().unwrap();
+        // let already_voted = tokens[10].clone().to_bool().unwrap();
+        ProxyBallotInfo {
+            start_time,
+            end_time,
+            total_voters,
+            progress,
+            is_finalized,
+            proposed_value,
+            contract_type,
+            creator,
+            memo,
+            can_be_finalized_now,
+            // already_voted,
+        }
+    }
+}
+
+impl ProxyBallotInfo {
+    fn email_text(&self) -> String {
+        format!(
+            "Voting Start Time: {}\n\
+            Voting End Time: {}\n\
+            Proposed New Proxy Address: {:?}\n\
+            Voting has Finished: {}\n\
+            Number of Votes Made: {}\n\
+            Ballot Creator: {:?}\n\
+            Memo: {}\n",
+            self.start_time,
+            self.end_time,
+            self.proposed_value,
+            self.is_finalized,
+            self.total_voters,
+            self.creator,
+            self.memo,
+        )
+    }
+}
+
+/// Returned by the V2 Emission Contract's `.getBallotInfo()` function:
+/// https://github.com/poanetwork/poa-network-consensus-contracts/blob/ec307069302fdf6647e8b1bdc13093960913b266/contracts/VotingToManageEmissionFunds.sol#L126
+#[derive(Clone, Debug)]
+pub struct EmissionBallotInfo {
+    pub creation_time: DateTime<Utc>,
+    pub start_time: DateTime<Utc>,
+    pub end_time: DateTime<Utc>,
+    pub is_canceled: bool,
+    pub is_finalized: bool,
+    pub creator: Address,
+    pub memo: String,
+    pub ammount: U256,
+    pub burn_votes: U256,
+    pub freeze_votes: U256,
+    pub send_votes: U256,
+    pub receiver: Address,
+}
+
+impl From<Vec<ethabi::Token>> for EmissionBallotInfo {
+    fn from(tokens: Vec<ethabi::Token>) -> Self {
+        let creation_time = {
+            let uint = tokens[0].clone().to_uint().unwrap();
+            u256_to_datetime(uint)
+        };
+        let start_time = {
+            let uint = tokens[1].clone().to_uint().unwrap();
+            u256_to_datetime(uint)
+        };
+        let end_time = {
+            let uint = tokens[2].clone().to_uint().unwrap();
+            u256_to_datetime(uint)
+        };
+        let is_canceled = tokens[3].clone().to_bool().unwrap();
+        let is_finalized = tokens[4].clone().to_bool().unwrap();
+        let creator = tokens[5].clone().to_address().unwrap();
+        let memo = tokens[6].clone().to_string().unwrap();
+        let ammount = tokens[7].clone().to_uint().unwrap();
```

Comment:
Could we convert the value in Wei to the value of POA coins? E.g., for `10441 POA` it displays `10441000000000000000000`. I think it's better to display `10441` instead. The amount also can be fractional.